src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +11,13 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String[] numbers = text.replaceAll("[^-0-9]", " ").trim().split("\\s+");
+        int[] numbersMultiply2 = new int[numbers.length];
+        int i = 0;
+        for (String numb : numbers) {
+            numbersMultiply2[i] = Integer.parseInt(numb) * 2;
+            i++;
+        }
```

Comment:
why are you using both `i` and for-each here?
also, consider renaming `numb` - the name is not informative